plugins/wallabag.koplugin/main.lua

```diff
@@ -397,30 +503,37 @@ The 'Synchronize remotely deleted files' option will remove local files that no
                 keep_menu_open = true,
                 callback = function()
                     UIManager:show(InfoMessage:new{
-                        text = T(_([[Wallabag is an open source read-it-later service. This plugin synchronizes with a Wallabag server.
+                        text = T(_([[wallabag is an open source read-it-later service. This plugin synchronizes with a wallabag server.
 
 More details: https://wallabag.org
 
-Downloads to folder: %1]]), BD.dirpath(filemanagerutil.abbreviate(self.directory)))
+Downloads to directory: %1]]), BD.dirpath(filemanagerutil.abbreviate(self.directory)))
```

Comment:
Sorry, I figured it would be better to be consistent and use _directory_ everywhere, instead of _folder_ and _directory_ interchangeably. I've changed these back

Reply:
Directory isn't used anywhere. 
<!-- Reviewable comment -OFRNpCo-1s4rr1CXWys:bypg749 -->
<!-- Sent from Reviewable.io -->